elastic/kibana
# [SecuritySolution] List Entities UI

This PR creates a UI component to list entities inside the Entity Store.

### What is included
 - Create `EntitiesList` component
 - Duplicate `MultiselectFilter` component
 - Display `EntitiesList` in the entity analytics dashboard
 - Use the `entityStoreEnabled` experimental flag 
 
### What is NOT included
 - Asset criticality
 - Source field
 - Risk score fields


![Screenshot 2024-09-20 at 15 27 23](https://github.com/user-attachments/assets/87295c76-a7d4-4303-b1ea-46d644bf21f4)



### How to test

1. Add some host/user data
    * Easiest is to use [elastic/security-data-generator](https://github.com/elastic/security-documents-generator)
2. Make sure to add `entityStoreEnabled` under `xpack.securitySolution.enableExperimental` in your `kibana.dev.yml`
3. In kibana dev tools or your terminal, call the `INIT` route for either `user` or `host`.
4. You should now see 2 transforms in kibana. Make sure to re-trigger them if needed so they process the documents.
5. Enable the experimental flag `entityStoreEnabled`
6. Go to entity analytics dashboard and you should see an populated entities page


Implements https://github.com/elastic/security-team/issues/10536

### Checklist
